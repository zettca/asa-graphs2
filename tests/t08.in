30 50
20
27 10 -2
30 12 2
26 11 -3
13 22 1
20 1 2
27 19 0
2 28 -2
23 14 -1
14 15 -1
1 13 -3
1 29 0
2 21 3
15 12 -3
19 16 -1
5 25 1
12 4 -3
24 13 1
4 11 -2
15 7 1
21 28 -2
12 6 0
5 14 2
28 6 -1
17 18 1
1 11 -2
8 3 0
28 7 3
21 9 1
30 26 0
16 17 -2
30 23 -3
24 2 3
9 27 1
3 12 -1
18 5 2
1 15 3
29 9 -1
24 14 3
5 28 0
6 17 3
23 16 0
2 9 -3
10 25 -1
22 20 1
27 7 0
26 2 2
25 4 0
2 14 2
25 14 -1
21 1 -3
