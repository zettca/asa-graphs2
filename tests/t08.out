2
U
U
-9
0
-6
-2
U
1
0
-11
-6
-1
-2
-3
1
-3
-2
2
0
U
0
U
U
-1
U
2
0
2
U
